TLM/TLM/Manager/Impl/RoutingManager.cs

```diff
@@ -2107,11 +2082,20 @@ bool laneChangesAllowed
                             totalIncomingLanes,
                             totalOutgoingLanes);
                     }
-                } // valid segment
 
-                if (nextSegmentId != prevSegmentId) {
-                    totalIncomingLanes += incomingVehicleLanes;
-                    totalOutgoingLanes += outgoingVehicleLanes;
+                    if (nextSegmentId != prevSegmentId) {
+                        totalIncomingLanes += incomingVehicleLanes;
+                        totalOutgoingLanes += outgoingVehicleLanes;
+                    }
+                } else {
+                    // invalid segment
+                    if (extendedLogRouting) {
+                        if (extendedLogRouting) {
```

Comment:
wat

Reply:
must have some merging error. Ill fix that.